deploy/examples/create-external-cluster-resources.py

```diff
@@ -312,7 +312,7 @@ def endpoint_dial(self, endpoint_str, timeout=3, cert=None):
                 # If verify is set to a path to a directory,
                 # the directory must have been processed using the c_rehash utility supplied with OpenSSL.
                 if prefix == "https" and cert and self._arg_parser.rgw_skip_tls:
-                    r = requests.head(ep, timeout=timeout, verify=False)
+                    r = requests.head(ep, timeout=timeout, verify=True)
```

Comment:
`self._arg_parser.rgw_skip_tls` if this is true then we don't need to verify the server's TLS certificate as the self-signed certificate will be passed.

At line 319 we are checking for it as it is default `true`.

Reply:
okay, so we don't need to check for validation in case of self-signed cert. In, Snyk it gives an error at this line.  I'm okay to close after @leseb  confirmation.  Thanks

Reply:
Like @parth-gr explained this is the intended behavior and we verify only when a certificate is passed.